Makefile

```diff
@@ -84,6 +84,7 @@ TARGETS += xfsprogs
 TARGETS += kernel
 TARGETS += drbd-pkg
 TARGETS += gasket-driver-pkg
+TARGETS += mellanox-ofed-pkg
```

Comment:
this file is generated, instead `.kres.yaml` should be updated with `make rekres` after that